example/index.html

```diff
@@ -69,90 +69,96 @@
-                 manifestPicker.append(option);
-             });
-         });
+        <script type="text/javascript">
+            var manifest;
+
+            $.get('http://dms-data.stanford.edu/data/manifests/BnF/jr903ng8662/manifest.json', function(manifest) {
+
+            manifest = manifest;
+
+            /* http://iiif.io/api/presentation/2.0/example/fixtures/24/manifest.json */
+            /* http://dms-data.stanford.edu/data/manifests/BnF/jr903ng8662/manifest.json */
+            viewer = manifestor({
+                manifest: manifest,
+                container: $('#example-container'),
+                perspective:  'detail',
+                canvasClass: 'canvas', //default set to 'canvas'
+                frameClass: 'frame', //default set to 'frame'
+                labelClass: 'label', //default set to 'label'
+                viewportPadding: {  // in detail view, make sure this area is clear
+                    top: 0,
+                    left: 10,
+                    right: 10,
+                    bottom: 10 // units in % of pixel height of viewport
+                },
+                stateUpdateCallback: function () {console.log('I have updated!')},
+                selectedCanvas: manifest.sequences[0].canvases[50]['@id']
+            });
+
+            key('h, left', function() {
+            viewer.previous();
+            });
+            /*          key('j, down') */ // Implement nearest Neighbour Search
+            /*          key('k, up') */   // Might be easiest if the lines were
+            // saved somewhere.
+            key('l, right', function() {
+            viewer.next();
+            });
+            });
+
+            // We can set events outside of the component.
+            $(window).on('resize', function() {
+            viewer.resize();
+            });
+
+            $('#manifestPicker').on('change', function() {
+            });
+
+            $('#mode').on('change', function(e) {
+            viewer.selectViewingMode(e.target[e.target.selectedIndex].value);
+            });
+
+            $('#scale').on('input', function() {
+            // must be a value between 0.5 and 2;
+            // input is between 50 and 200, so
+            // we must convert.
+            viewer.updateThumbSize($(this).val()*(1/100));
+            });
+
+            key('shift+j', function() {
+            viewer.selectPerspective('detail');
+            console.log('shifting to detail perspective');
+            });
+            key('shift+k', function() {
+            viewer.selectPerspective('overview');
+            console.log('shifting to overview perspective');
+            });
+            key('m', cycleViewingModes);
+
+            var currentMode = 'individuals';
+
+            function cycleViewingModes() {
+            var newMode;
+
+            if (currentMode === 'individuals') {
+            newMode = 'paged';
```

Comment:
Good question. I'll tidy this back up. It likely has something to do with my editor's settings.